testing/e2e/app/routes/Table/spec.ts

```diff
@@ -4,9 +4,10 @@ test.describe('Table sorting', () => {
   test('should work with keyboard', async ({ page }) => {
     await page.goto('/Table');
 
-    const firstColumnCells = page.locator('[role="cell"]:first-child');
+    const firstColumnCells = page.locator('[role="cell"]:nth-child(2)');
```

Comment:
There shouldn't be any big changes in existing tests. Keep the old component and test unchanged, and use search-params to enable additional features for new tests.

Reply:
Okay, will fix.